frontend/catalyst/jax_primitives.py

```diff
@@ -76,7 +76,7 @@
 from catalyst.utils.extra_bindings import FromElementsOp, TensorExtractOp
 from catalyst.utils.types import convert_shaped_arrays_to_tensors
 
-# pylint: disable=unused-argument,abstract-method,too-many-lines
+# pylint: disable=unused-argument,too-many-lines
```

Comment:
It's quite annoying because I've noticed pylint doesn't act the same in all environments. Just ran `pylint frontend` and `abstract-method` comes up when removed, so let's leave it.
```suggestion
# pylint: disable=unused-argument,abstract-method,too-many-lines
```